packages/gasket-plugin-happyfeet/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "gasket-plugin-happyfeet",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest"
+  },
+  "keywords": [],
```

You need to include the [repository](https://docs.npmjs.com/cli/v6/configuring-npm/package-json?v=true#repository) and [publishConfig](https://docs.npmjs.com/cli/v6/configuring-npm/package-json?v=true#publishconfig) properties:

```suggestion
  "repository": {
    "type": "git",
    "url": "git+ssh://git@github.com/godaddy/gasket.git"
  },
  "publishConfig": {
    "access": "public"
  },
  "keywords": [],
```

Adding some keywords won't hurt as well.